File: 1061.js
const { Console } = require('console');

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var dia_i = lines.shift().split(" ");
var hora_i = lines.shift().split(" ");
var dia_f = lines.shift().split(" ");
var hora_f = lines.shift().split(" ");
var di = Number(dia_i[1]); var df = Number(dia_f[1]);
var hi = Number(hora_i[0]); var mi = Number(hora_i[2]); var si = Number(hora_i[4]);
var hf = Number(hora_f[0]); var mf = Number(hora_f[2]); var sf = Number(hora_f[4]);
var minuto_seg = 60
var hora_seg = minuto_seg * 60
var dia_seg = hora_seg * 24
var i = si + mi*minuto_seg + hi*hora_seg + di*dia_seg
var f = sf + mf*minuto_seg + hf*hora_seg + df*dia_seg
if (i < f){
    var tempo = f - i
    var dias = Number(tempo/dia_seg)
    var tempo = tempo%dia_seg
    var horas = Number(tempo/hora_seg)
    var tempo = tempo%hora_seg
    var minutos = Number(tempo/minuto_seg)
    var tempo = tempo%minuto_seg
    var segundos = tempo}
W = dias
X = horas
Y = minutos
Z = segundos
console.log(`${parseInt(W)} dia(s)
${parseInt(X)} hora(s)
${parseInt(Y)} minuto(s)
${parseInt(Z)} segundo(s)`)

File: 1061.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
    int di, df, mi, si, hi, hf, mf, sf, dias, horas, minutos, segundos;
    string d;
    cin >> d >> di;
    cin >> hi >> d >> mi >> d >> si; 
    cin >> d >> df;
    cin >> hf >> d >> mf >> d >> sf;

    int minuto_seg = 60;
    int hora_seg = minuto_seg * 60;
    int dia_seg = hora_seg * 24;
    int i = si + mi*minuto_seg + hi*hora_seg + di*dia_seg;
    int f = sf + mf*minuto_seg + hf*hora_seg + df*dia_seg;
    if(i < f){
        int tempo = f - i;
        dias = tempo/dia_seg;
        tempo = tempo%dia_seg;
        horas = tempo/hora_seg;
        tempo = tempo%hora_seg;
        minutos = tempo/minuto_seg;
        tempo = tempo%minuto_seg;
        segundos = tempo;
    }
    int W = dias;
    int X = horas;
    int Y = minutos;
    int Z = segundos;

    cout << W << " dia(s)\n" << X << " hora(s)\n" << Y << " minuto(s)\n" << Z << " segundo(s)\n";
    return 0;
}
